### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by cmcuza**
This PR adds the README.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,47 @@
...
+
+# Installation
+TerseTS can be compiled and cross-compiled from source:
+1. Install the latest version of [Zig](https://ziglang.org/)
+2. Build TerseTS for development in `Debug` mode using Zig, e.g.,:
+   - Linux: `zig build -Dtarget=x86_64-linux`
+   - macOS: `zig build -Dtarget=aarch64-macos`
+   - Microsoft Windows: `zig build -Dtarget=x86_64-windows`
+3. Build TerseTS for deployment in `ReleaseFast`, `ReleaseSafe`, and `ReleaseSmall` mode using Zig, e.g.,:
+   - Linux: `zig build -Dtarget=x86_64-linux`
```
Comment (skejserjensen) [inline README.md:14]: One of the release build types should part of the example.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,47 @@
...
+   - macOS: `zig build -Dtarget=aarch64-macos`
+   - Microsoft Windows: `zig build -Dtarget=x86_64-windows`
+
+# Usage
```
Comment (skejserjensen) [inline README.md:18]: Maybe remove or replace this section with a TODO until the code for the user interface is committed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,47 @@
...
+TerseTS's API is inspired by [libpressio](https://github.com/robertu94/libpressio) and currently provides these methods:
+```zig
+export fn compress(input: Input, output: *Output) i32;
+```
+
+The data to be compressed and the compression method to use is provided as an instance of `Input`, the compressed data is written to an instance of `Output`, and if an error occurs a non-zero integer is returned. Thus, only the instance of `Input` has to be changed to use a completely compression method or change the configuration of a compression method.
+
+
+# Bindings
+TerseTS provides a C-API that is designed to be simple to wrap. Currently, TerseTS includes bindings for the current languages which can be used without installation of any dependencies:
```
Comment (skejserjensen) [inline README.md:28]: Maybe replace "languages" with "programming languages".

File: README.md [hunk 4]
```diff
@@ -1,2 +1,47 @@
...
+- [Zig](tersets/capi.zig)
+- [C](tersets/capi.zig)
+- [C++](tersets/capi.zig) 
+- [Python](bindings/tersets.py) using [ctypes](https://docs.python.org/3/library/ctypes.html)
+
+# Links
```
Comment (skejserjensen) [inline README.md:34]: Maybe remove this section.

**Review by skejserjensen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cmcuza) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (cmcuza) [reply README.md:18]: Fixed.

File: README.md (hunk 3 above)
Comment (cmcuza) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (cmcuza) [reply README.md]: Fixed.

**Review by skejserjensen — APPROVED** [review]

**Pull request merged**